### PR TITLE
Use variables with descriptive names rather than literal values as pa…

### PR DESCRIPTION
**Pull request by tdunn**
…rameters.

@lsmallwood - Please review and merge. Thank you.

### CODE REVIEW

File: ArcGISRuntimeSDKQt_CppSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.cpp [hunk 1]
```diff
@@ -95,7 +95,9 @@ void DeleteFeaturesFeatureService::connectSignals()
 
         //! [DeleteFeaturesFeatureService identify feature]
         // call identify on the map view
-        m_mapView->identifyLayer(m_featureLayer, mouseEvent.x(), mouseEvent.y(), 5, IdentifyReturns::GeoElementsOnly, 1);
+        double tolerance = 5;
```
Comment (lsmallwood) [inline ArcGISRuntimeSDKQt_CppSamples/EditData/DeleteFeaturesFeatureService/DeleteFeaturesFeatureService.cpp]: as this is a double, should possibly make it 5.0

**Comment by lsmallwood** [comment]
@tdunn - looks good. Just the one comment

**Comment by tdunn** [comment]
@lsmallwood - Back to you, thanks.

**Comment by lsmallwood** [comment]
That's' great - thanks for making those changes @tdunn

**Pull request merged**